### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by matto2** [webcomponents/webcomponents.org#1249]
[Origin page](https://www.webcomponents.org/element/BrightspaceUI/checkbox)

Looks cool- but I think the unchecked state has insufficient contrast for accessibility. 

(I see huge value in web components with accessibility built in as a core requirement)
